hash_practice/exercises.py

```diff
@@ -26,4 +55,4 @@ def valid_sudoku(table):
         Space Complexity: ?
     """
     pass
-
+print(top_k_frequent_elements([1, 2, 2, 2, 3, 3, 3], 2))
```

Comment:
Make sure to remove testing code like this before checking in, just to keep the code clean for style purposes.